### PR TITLE
Get direct annotation functionality

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

**Comment by dsavchenko** [comment]
The failure in "Executing notebook" is unrelated, I opened an [issue](https://github.com/oda-hub/oda_api/issues/265)

File: oda_api/ontology_helper.py [hunk 1]
```diff
@@ -400,6 +400,19 @@ def get_oda_label(self, param_uri):
         
         return label
 
+    def get_oda_metadata(self, param_uri, metadata):
```
[the remark this answers is not in the document]
Comment (burnout87) [reply oda_api/ontology_helper.py:403]: `get_direct_annotation_metadata` ?

[the remark this answers is not in the document]
Comment (dsavchenko) [reply oda_api/ontology_helper.py:403]: Well, it's used for 'metadata' here, but basically, graph don't know of what 'metadata' means here, the method can be used to obtain any direct annotation of the parameter

**Comment by dsavchenko** [comment]
Any ideas of why "LiveAPI tests" hangs? 
I see the same in my PR

**Comment by burnout87** [comment]
> Any ideas of why "LiveAPI tests" hangs? I see the same in my PR
> 
> UPD: it hangs waiting for the job created in Cell [7] of `doc/source/user_guide/Show_and_Save_Products.ipynb` notebook to get ready, but it seems this job (`2c8b65d7f62a0d7c`) is stuck in the `submitted` state on prod instance.

yes, that is the reason why. I also have some jobs that are stuck

**Comment by burnout87** [comment]
> Any ideas of why "LiveAPI tests" hangs? I see the same in my PR
> 

Tests now pass

**Comment by burnout87** [comment]
there is an issue ongoing with the `setuptools` module

https://github.com/pypa/setuptools/issues/4519

**Review by volodymyrss — APPROVED** [review]
- [x] Please don't forget to make an issue to add a doc.

**Comment by burnout87** [comment]
> there is an issue ongoing with the `setuptools` module
> 
> [pypa/setuptools#4519](https://github.com/pypa/setuptools/issues/4519)

It is resolved now, but no apparent actions from my side :/

**Review by okolo — APPROVED** [review]

**Review by dsavchenko — APPROVED** [review]

**Pull request merged**